storybook/src/admin/form/FieldContainer.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import { FieldSet, TextField } from "@comet/admin";
+import * as React from "react";
+import { Form } from "react-final-form";
+
+export default {
+    title: "@comet/admin/form",
+};
+
+export const FieldContainer = () => {
+    return (
+        <Form
+            onSubmit={() => {
+                // do nothing
+            }}
+            render={({ handleSubmit }) => (
+                <form onSubmit={handleSubmit}>
+                    <FieldSet title="HORIZONTAL (Full-width)">
+                        <TextField name="textFieldWithLabelOne" label="TextField label" variant="horizontal" fullWidth />
+                        <TextField name="textFieldWithLabelTwo" label="TextField label" variant="horizontal" fullWidth />
+                        <TextField name="testWithoutLabel" variant="horizontal" placeholder="TextField without label" fullWidth />
+                    </FieldSet>
+                    <FieldSet title="VERTICAL (Full-width)">
+                        <TextField name="textFieldWithLabelOne" label="TextField label" variant="vertical" fullWidth />
+                        <TextField name="textFieldWithLabelTwo" label="TextField label" variant="vertical" fullWidth />
+                        <TextField name="testWithoutLabel" variant="vertical" placeholder="TextField without label" fullWidth />
+                    </FieldSet>
+                    <FieldSet title="HORIZONTAL (auto-width)">
+                        <TextField name="textFieldWithLabelOne" label="TextField label" variant="horizontal" />
+                        <TextField name="textFieldWithLabelTwo" label="TextField label" variant="horizontal" />
+                        <TextField name="testWithoutLabel" variant="horizontal" placeholder="TextField without label" />
+                    </FieldSet>
+                    <FieldSet title="VERTICAL (auto-width)">
+                        {/* FieldSet is not made to include non-full-width fields. Adding this `div` removes the flex-behaviour of FieldSet, allowing this example. */}
```

Comment:
If it isn't made for it, why add this example at all?

Reply:
For debugging when multiple non-full-width fields are used inside a container. 

Arguably this is actually a bug and `FieldSet` should not set `display: flex`... I'll probably fix this in V8.

Reply:
I created a task and mentioned it in a TODO comment: [Mention fix-task](https://github.com/vivid-planet/comet/pull/2872/commits/41396f0003bdb82c7f6668a787f334fe94f34ea8)